create addEventListener

GreatDeveloper66 opened the pull request:
<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

GreatDeveloper66 committed:
create addEventListener

GreatDeveloper66 committed:
Change title to lower case

RandellDawson commented:
@GreatDeveloper66 You are still failing the Travis build.  

- The actual file name should be `index.md` as was first mentioned.  Currently, the file is named `addeventlistener`.  Also, you are missing three backticks on line one of the file.  Also, please change your title to `addEventListener` in the frontmatter block.

Thank you.

GreatDeveloper66 committed:
Rename file to index.md

Update and rename addeventlistener file to index.md. Also change front matter block to meet style guide

RandellDawson commented:
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉
